client/src/components/player/VolumeControls.js

```diff
@@ -32,23 +32,128 @@ const ProgressBar = styled.div`
   width: 100%;
 `;
 
-function VolumeControls() {
-  return (
-    <VolumeControlsWrapper>
-      <VolumeBar>
-        <PlayerButton title="Enable Repeat" color="white">
-          <i className="material-icons md-20">volume_up</i>
-        </PlayerButton>
-        <ProgressBar>
-          <PlayerPlaybackProgress>
-            <PlayerPlaybackProgressBg>
-              <PlayerPlaybackProgressBgCharge progress={'7'} />
-            </PlayerPlaybackProgressBg>
-          </PlayerPlaybackProgress>
-        </ProgressBar>
-      </VolumeBar>
-    </VolumeControlsWrapper>
-  );
+class VolumeControls extends Component {
```

Comment:
you could use here function component by using hooks, its a way to go now. 
https://reactjs.org/docs/hooks-intro.html